litmus-portal/frontend/src/components/Header/styles.ts

```diff
@@ -28,6 +28,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     paddingBottom: theme.spacing(0),
     marginLeft: theme.spacing(5),
     paddingleft: theme.spacing(5),
+    color: theme.palette.userNameTextColor,
```

Comment:
Can't we pass the color from parent, Instead of defining it in component

Reply:
yes, we can but most places we are using different values for a variable key like color, backgroundColor. So, wherever possible to declare and use a single value have modified them, others that use a unique value, I haven't modified them for now.

Reply:
okay